libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -97,6 +98,26 @@ const AP_Param::GroupInfo AP_OSD_Screen::var_info[] = {
     // @Path: AP_OSD_Setting.cpp
     AP_SUBGROUPINFO(home, "HOME", 14, AP_OSD_Screen, AP_OSD_Setting),
 
+    // @Group: BLHTEMP
```

Comment:
this needs #ifdef HAVE_AP_BLHELI_SUPPORT around the blheli items

Reply:
Done